.gitignore

```diff
@@ -10,9 +10,9 @@ pubspec.lock
 /coverage/
 
 # generated assets in test fixtures
-/test/fixtures/coverage/browser/coverage/
-/test/fixtures/coverage/functional_test/coverage
-/test/fixtures/coverage/browser_needs_pub_serve/coverage/
-/test/fixtures/coverage/non_test_file/coverage/
-/test/fixtures/coverage/vm/coverage/
-/test/fixtures/docs/docs/doc/api/
+/test_fixtures/coverage/browser/coverage/
+/test_fixtures/coverage/functional_test/coverage
+/test_fixtures/coverage/browser_needs_pub_serve/coverage/
+/test_fixtures/coverage/non_test_file/coverage/
+/test_fixtures/coverage/vm/coverage/
+/test_fixtures/docs/docs/doc/api/
```

Comment:
I moved `test/fixtures/` to `test_fixtures/` so that there's no longer a conflict between the dart_dev project and the fixtures.